<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttributeViewWidgetClass</class>
 <widget class="QWidget" name="AttributeViewWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AttributeViewWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>61</height>
      </size>
     </property>
     <widget class="QPushButton" name="ctrl_btnEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>启用编辑</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ctrl_btnAddFeature">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>新增要素</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ctrl_btnDelFeature">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>删除要素</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ctrl_btnAddAttri">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>20</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>添加字段</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ctrl_btnDelAttri">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>20</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>删除字段</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="tableView">
     <property name="autoScrollMargin">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
